api/models/comment.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const commentSchema = mongoose.Schema({
+    _id: mongoose.Schema.Types.ObjectId,
+    issue: { type: mongoose.Schema.Types.ObjectId, ref: 'Issue' },
+    content: { type: String, required: true }
```

Comment:
Can you check if there is a length limit for `String` type in mongo. If there is do you think that limit is reasonoable for comment content?